esti/ops/docker-compose.yaml

```diff
@@ -11,6 +11,7 @@ services:
       - lakefs-app:/app:ro
     environment:
       - LAKEFS_AUTH_ENCRYPT_SECRET_KEY=some random secret string
+      - LAKEFS_DATABASE_TYPE=${LAKEFS_DATABASE_TYPE:-postgres}
       - LAKEFS_DATABASE_CONNECTION_STRING=postgres://lakefs:lakefs@postgres/postgres?sslmode=disable
```

Comment:
I think this should be removed from the lakefs docker compose - and added explicitly in jobs where we need this env var

Reply:
Exactly - we should add it only to the migrate test

Reply:
Done